e2e_testing/xfail_sets.py

```diff
@@ -316,6 +316,7 @@
     "RsubFloatModule_noalpha_basic",
     "RsubIntModule_basic",
     "RsubIntModule_noalpha_basic",
+    "RsubInt0d_NumToTensor_Module",
```

Comment:
This should be the name of the Python function that gets registered as the test. In this case, it should be: `RsubInt0d_NumToTensor_Module_basic`